internal/services/network/web_application_firewall_policy_resource.go

```diff
@@ -353,23 +371,34 @@ func resourceWebApplicationFirewallPolicy() *pluginsdk.Resource {
 							}, false),
 							Default: string(webapplicationfirewallpolicies.WebApplicationFirewallModePrevention),
 						},
+
 						"request_body_check": {
 							Type:     pluginsdk.TypeBool,
 							Optional: true,
 							Default:  true,
 						},
+
 						"file_upload_limit_in_mb": {
 							Type:         pluginsdk.TypeInt,
 							Optional:     true,
 							ValidateFunc: validation.IntBetween(1, 4000),
 							Default:      100,
 						},
+
 						"max_request_body_size_in_kb": {
 							Type:         pluginsdk.TypeInt,
 							Optional:     true,
 							ValidateFunc: validation.IntBetween(8, 2000),
 							Default:      128,
 						},
+
+						"request_body_inspect_limit_in_kb": {
+							Type:         pluginsdk.TypeInt,
+							Optional:     true,
+							Default:      128,
```

Comment:
Is this the default the API uses?

Reply:
the swagger does not give this value, but the API always repsonses `128` when tested locally.